nested/mssql.json

```diff
@@ -0,0 +1,63 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "moodleCommon": {
+            "metadata": {
+                "description": "Common Moodle values"
+            },
+            "type": "object"
+        }
+    },
+    "resources": [
+        {
+            "apiVersion": "2015-05-01-preview",
+            "kind": "",
+            "location": "[resourceGroup().location]",
+            "name": "[parameters('moodleCommon').serverName]",
+            "properties": {
+                "administratorLogin": "[parameters('moodleCommon').dbLogin]",
+                "administratorLoginPassword": "[parameters('moodleCommon').dbLoginPassword]",
+		"sslEnforcement": "[parameters('moodleCommon').sslEnforcement]",
+                "storageMB": "[parameters('moodleCommon').skuSizeMB]",
+                "version": "[parameters('moodleCommon').mssqlVersion]"
+            },
+            "resources": [
+                {
+                    "apiVersion": "2014-04-01",
+                    "dependsOn": [
+                        "[concat('Microsoft.Sql/servers/', parameters('moodleCommon').serverName)]"
+                    ],
+                    "location": "[resourceGroup().location]",
+                    "name": "[parameters('moodleCommon').firewallRuleName]",
+                    "properties": {
+                        "startIpAddress": "0.0.0.0",
+                        "endIpAddress": "255.255.255.255"
+                    },
+                    "type": "firewallrules"
+                }
+            ],
+            "sku": {
```

Comment:
Same here. I don't see this property specified in the REST API reference spec page. Could you point me to the spec page you used when including this entry? If this is not required or used by ARM, I think we should remove this to avoid any confusion.